hw/xbox/xid-sdl.c

```diff
@@ -464,6 +466,59 @@ static void usb_xid_class_initfn(ObjectClass *klass, void *data)
     uc->handle_attach  = usb_desc_attach;
 }
 
+static HapticEmulationMode detect_haptic_emulation_mode(SDL_Haptic *haptic) {
+    if (SDL_HapticQuery(haptic) & SDL_HAPTIC_LEFTRIGHT) {
+        return EMU_HAPTIC_LEFT_RIGHT;
+    }
+
+    return EMU_UNSUPPORTED;
+}
+
+static int haptic_init(USBXIDState *s) {
+    if (SDL_HapticOpened(s->device_index)) { return 0; }
```

Comment:
Given https://github.com/xqemu/xqemu/pull/197#discussion_r255591397, this still implies we should return an error along the lines of "Haptic device is already in use" (or some more user-friendly equivalent).

Reply:
Good call, and thanks for the clarification on https://github.com/xqemu/xqemu/pull/197#discussion_r255591397, I clearly miss-understood what the comment was mentioning, and now I can read up on some QOM docs!